dist/tools/ethos/start_network.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+trap "cleanup" EXIT #SIGINT SIGTERM
+
+create_tap() {
+    ip tuntap add ${TAP} mode tap user ${USER}
+    sysctl -w net.ipv6.conf.${TAP}.forwarding=1
```

Comment:
http://www.tldp.org/HOWTO/Linux+IPv6-HOWTO/proc-sys-net-ipv6..html

It is not recommended to have different forwarding settings for individual interfaces.